docs/development.rst

```diff
@@ -388,6 +388,14 @@ recommended process for generating a new release is described here.
 
         python setup.py sdist upload
 
+10. Ensure that a built Docker image is tagged on
+    `Girder's Docker Hub page <https://hub.docker.com/r/girder/girder/tags/>`_.
+    Such a build can be added by following the instructions for
+    `adding and running a new build <https://docs.docker.com/docker-hub/builds/#add-and-run-a-new-build>`_.
```

Comment:
It sounds like clicking the tag button on dockerhub is equivalent to pull/tag/push from the terminal.  It seems like there is more room for error doing it on the terminal though if, for example, you forgot to pull or accidentally built to the tag locally.